lib/modules/platform/gitea/index.ts

```diff
@@ -600,6 +601,19 @@ const platform: Platform = {
       prUpdateParams.base = targetBranch;
     }
 
+    /**
+     * Update PR labels.
+     * In the Gitea API, labels are replaced on each update if the field is present.
+     * If the field is not present (i.e., undefined), labels aren't updated.
+     * However, the labels array must contain label IDs instead of names,
+     * so a lookup is performed to fetch the details (including the ID) of each label.
+     */
+    if (Array.isArray(labels)) {
+      prUpdateParams.labels = (await map(labels, lookupLabelByName)).filter(
```

Comment:
we should log a warning when a label isn't found, because renovate will stop updating labels because of debug data mismatch.

renovate thinks labels were changed by someone else.